### ISSUE TITLE
Support other chromium browsers.

### ISSUE THREAD

**Issue by blfunex** [porsager/wright#37]
After using flems for a bit, I'm hooked to the workflow. but I exclusively use Brave. so no Chrome for me, but Wright complains:

```
Can't find Chrome at:
"C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe"
If Chrome is installed somewhere else, set the environment variable CHROME_PATH
```

now I have to install **google** on my computer.

I hope it can be fixed 😄

Thanks for all your work @porsager, you are an inspiration.

**Comment by porsager** [comment]
Hi. Thank you for your kind words ;)

I've just tried the same on my computer, and if brave follows the chrome devtools api and startup arguments I think it should work, but for some reason `fs.existsSync('C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe')` returns false.. I've tried from a regular command prompt and also an elevated one with no difference. 

I'm a bit short of time these days, but you can disable using HMR and just add `--browser false` if you are using `wright@next`.

If you want to dig into the issue above maybe try to figure out why the exists call returns false for the brave.exe path.

**Comment by blfunex** [comment]
Thanks I will pass, I'm a bit short of time also, so I installed chrome for this once, I will happily wait until this is fixed.

**Comment by blfunex** [comment]
I tried with node's REPL
```
fs.existsSync('C:\\Program Files (x86)\\BraveSoftware\\Brave-Browser\\Application\\brave.exe')
```
and it works 😆

**Comment by blfunex** [comment]
notice the `\\`

also my node has no administrator privileges

**Comment by porsager** [comment]
Hah.. Just had a few minutes to dig deeper, and it was a simple issue of a trailing whitespace from the windows env. set variable :D

I'll push a fix with `trim()` added and release a new 2.0 beta ;)
